src/datasources/db/v2/postgresql-logger.service.ts

```diff
@@ -0,0 +1,89 @@
+import type { ILoggingService } from '@/logging/logging.interface';
+import { Injectable } from '@nestjs/common';
+import {
+  AbstractLogger,
+  type LoggerOptions,
+  type LogLevel,
+  type LogMessage,
+} from 'typeorm';
+
+@Injectable()
+export class PostgresqlLogger extends AbstractLogger {
+  public constructor(
+    public options?: LoggerOptions,
+    private readonly loggingService?: ILoggingService,
+  ) {
+    super(options);
+  }
+  /**
+   * Write log to specific output.
+   *
+   * This is taken from the TypeOrm default implementation, check out {@link https://orkhan.gitbook.io/typeorm/docs/logging}
+   */
+  protected writeLog(
```

Comment:
This is a follow on to [@hectorgomezv's comment regarding direct usage of `console`](https://github.com/safe-global/safe-client-gateway/pull/1988/files#r1789965643). If we inject the `loggingService` as per my above comment, we could/should use it instead of `console.log`, for example.

Reply:
The `loggingService` is injected and used in `logQuery` method. Since this was taken from the default implementation I didn't change it. Will use logger here as well.

Reply:
Done https://github.com/safe-global/safe-client-gateway/pull/1988/commits/2ea0e5d8a752fd31cff745ff3490d8fe8134ddc7